src/other/precompile.jl

```diff
@@ -1579,9 +1579,9 @@ function precompile(all=false)
         Base.precompile(Tuple{Reduce{typeof(max), Nothing, Nothing},Vector{Int},GroupedDataFrame{DataFrame}})
         Base.precompile(Tuple{Type{OnCol},Vector{String},Vararg{AbstractVector{T} where T, N} where N})
 
-        combine(groupby(DataFrame(), []), identity)
-        innerjoin(DataFrame(v1=[]), DataFrame(v1=[]), on=:v1)
-        outerjoin(DataFrame(v1=[]), DataFrame(v1=[]), on=:v1)
+        combine(groupby(DataFrame(v1=Int[]), :v1), identity, :v1 => sum)
+        innerjoin(DataFrame(v1=Int[]), DataFrame(v1=Int[]), on=:v1)
+        outerjoin(DataFrame(v1=Int[]), DataFrame(v1=Int[]), on=:v1)
     end
```

Comment:
If you do this, maybe also include a `String` and a `PooledArray{String}` column? Could also use e.g. `:v1 => identity` to cover the fallback.

Reply:
I went for an almost full sweep then.